src/Seq.Input.RabbitMQ/RabbitMQListener.cs

```diff
@@ -1,21 +1,24 @@
-﻿using System;
+using System;
+using System.Net.Security;
 using RabbitMQ.Client;
 using RabbitMQ.Client.Events;
 
 namespace Seq.Input.RabbitMQ
 {
-    class RabbitMQListener : IDisposable
+    public class RabbitMQListener : IDisposable
```

Comment:
Nit: public isn't needed here

Reply:
Added public for my local testing. My unittest contains credentials and will not be pushed to github ;)